account_asset_management/views/account_move.xml

```diff
@@ -4,6 +4,36 @@
         <field name="name">account.move.form.account.asset.management</field>
         <field name="model">account.move</field>
         <field name="inherit_id" ref="account.view_move_form" />
+        <field name="arch" type="xml">
+            <xpath
+                expr="//field[@name='invoice_line_ids']/tree/field[@name='quantity']"
+                position="before"
+            >
+                <field
+                    name="asset_profile_id"
+                    domain="[('company_id','=', parent.company_id)]"
+                    attrs="{'column_invisible': [('parent.type', 'not in', ('in_invoice', 'in_refund'))]}"
+                    optional="show"
+                />
+            </xpath>
+            <xpath
+                expr="//notebook//field[@name='line_ids']/tree/field[@name='date_maturity']"
+                position="after"
+            >
+                <field
+                    name="asset_profile_id"
+                    domain="[('company_id','=', parent.company_id)]"
+                    optional="show"
+                />
+            </xpath>
+        </field>
+    </record>
+    <!--    Split view to avoid unnecessary compute asset_count (attrs is evaluated although groups is set -->
+    <record id="view_move_form_managers" model="ir.ui.view">
+        <field name="name">account.move.form.account.asset.management.managers</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account_asset_management.view_move_form" />
+        <field name="groups_id" eval="[(4, ref('account.group_account_manager'))]" />
```

Comment:
There's something strange here.

We have 3 groups with access to these features: https://github.com/OCA/account-financial-tools/blob/98e88539085fe4f49d48980ca9a35a00b9127a46/account_asset_management/security/ir.model.access.csv#L8-L10

Well, those permissions are a bit redundant because they assign the same permissions to all 3 groups and `account.group_account_manager` inherits from `account.group_account_invoice`, so the permissions to `account.group_account_manager` are unnecessary (they are redundant). But that's another issue.

The point here is that IMHO it doesn't make much sense to give access to things that they won't see, so I think the correct line would be the one from https://github.com/OCA/account-financial-tools/pull/1198:

```suggestion
        <field name="groups_id" eval="[(4, ref('account.group_account_invoice')), (4, ref('account.group_account_user'))]" />
```

Otherwise, probably we would have to change some security rules.

Or maybe I'm just wrong and this isn't like I say because the only ones that should access these features are managers... well, TBH I have no idea what this module does 😆